chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -187,7 +187,10 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
                             eventName: BroadcastEvent.StartChatSkippingChatButtonRendering,
                         };
                         BroadcastService.postMessage(chatStartedSkippingChatButtonRendering);
-                        setPreChatAndInitiateChat(chatSDK, props.chatConfig, props.getAuthToken, dispatch, setAdapter);
+                        const optionalParams = (!isUndefinedOrEmpty(state.domainStates?.liveChatContext) && state.appStates.conversationState === ConversationState.Active) ?
+                            { liveChatContext: state.domainStates?.liveChatContext } :
```

Comment:
Wouldn't it get reset at line https://github.com/microsoft/omnichannel-chat-widget/blob/c9a643a4fb5d8c219f3b408b52b7e4f5d8919199/chat-widget/src/components/livechatwidget/common/startChat.ts#L75 ?

Reply:
Fixed